spec/zizia/csv_template_spec.rb

```diff
@@ -7,9 +7,12 @@
 
   before do
     Zizia.config.metadata_mapper_class = Zizia::HyraxBasicMetadataMapper
+    # rubocop:disable RSpec/AnyInstance
+    allow_any_instance_of(Zizia::HyraxBasicMetadataMapper).to receive(:fields).and_return([:based_near, :resource_type, :description])
+    # rubocop:enable RSpec/AnyInstance
   end
 
   it 'returns a CSV string based on the headers in the Hyrax mapper' do
-    expect(csv_template.to_s).to eq(Zizia::HyraxBasicMetadataMapper.new.fields.join(','))
+    expect(csv_template.to_s).to eq("location,resource type,abstract or summary")
```

Comment:
space after comma?

Reply:
no, it's a fake csv, so the commas are delimiters. Otherwise, you'd have random spaces in front of your headers

Reply:
oh wait, this is in a csv, nm